internal/ingress/annotations/annotations.go

```diff
@@ -46,15 +46,15 @@ const (
 	DefaultIngressClass = "kong"
 )
 
-func validIngress(ingressAnnotationValue, ingressClass string) bool {
+func validIngress(ingressAnnotationValue, ingressClass string, skipEmptyAnnotation bool) bool {
```

Comment:
Can you rename the variable to `skipClasslessIngress`?
Ingress class is a formalized concept in Ingress v1 and we would like to reuse parts of this PR for that as well.